app/models.py

```diff
@@ -49,3 +53,16 @@ class Tunnel(db.Model):
 
     def __repr__(self):
         return "<Tunnel {} {}>".format(self.subdomain, self.job_id)
+
+
+class TokenBlacklist(db.Model):
```

Comment:
What happened to the idea of just changing the User's uuid to revoke all of their tokens, and not needing to store a blacklist?

Reply:
I added the blacklist which is really just a list of old uuids and when they expire. It makes it so we can add this 
```@jwt.token_in_blacklist_loader
    def check_if_token_revoked(decoded_token):
        return is_token_revoked(decoded_token)```

Reply:
But why?  Just check if their UUID matches a user and if not it's not a valid token, and it's unauthorized.

Reply:
fair enough